mobile/src/auth/pin-code-storage.ts

```diff
@@ -0,0 +1,48 @@
+/******************************************************************************
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import SInfo from 'react-native-sensitive-info';
+import Storage from './storage';
+
+//============================================================================
+export class PinCodeStorage {
+    private static options = {
+        keychainService: Storage.keychain,
+        sharedPreferencesName: Storage.preferences,
+    };
+
+    private pin: string | null = null;
+
```

Comment:
Why don't you use Nullable which is used in the rest of the project?

Reply:
@YaroTs stage 5, acceptance.